onnx/backend/base.py

```diff
@@ -60,7 +60,7 @@ class BackendRep:
     BackendRep to retrieve the corresponding results.
     """
 
-    def run(self, inputs: Any, **kwargs: Any) -> Tuple[Any, ...]:
+    def run(self, inputs: Any, **kwargs: Any) -> Tuple[Any, ...]:  # type: ignore
```

Comment:
I had to ignore this line in `mypy` to pass linter check. It was throwing
`onnx/backend/base.py:63:5: error: Missing return statement  [empty-body]`
Is this a known issue? I did not make changes to this file. If you want me to fix it please let me know.

Reply:
I see Jacky has a PR: https://github.com/onnx/onnx/pull/4643

Reply:
Just merged. Updating the latest main branch in this PR should fix this issue. Thanks!